web/src/assets/Logo.svg

```diff
@@ -0,0 +1,18 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="198.138" height="31.324" viewBox="0 0 198.138 31.324">
```

Comment:
This file will need renaming to `logo.svg`

Reply:
I forgot to push ..

Reply:
Its not visible in the changes